frontend/javascripts/oxalis/model/accessors/flycam_accessor.ts

```diff
@@ -307,7 +312,7 @@ export function getValidTaskZoomRange(
   state: OxalisState,
   respectRestriction: boolean = false,
 ): [number, number] {
-  const defaultRange = [userSettings.zoom.minimum, Infinity];
+  const defaultRange = [baseDatasetViewConfiguration.zoom.minimum, Infinity];
```

Comment:
Do you know why typescript did not complain about this, before?

Reply:
If I remember correctly, both code versions are fine value-wise (i.e., they are set to the same value), but the old one caused a cyclic dependencies, about which TS doesn't care :)

Reply:
Ah, I searched for `zoom` in the `user_settings.schema` and couldn't find it, but there's a `...baseDatasetViewConfiguration,` in there that I missed :+1: